.html

```diff
@@ -0,0 +1,10 @@
+
+<html>
+<body>
+
+<h1 style="background-color: ;">Hello Microverse!</h1>
+
+
+</body>
+</html>
+
```

Comment:
Great job @TayyabKM 🤝

- [ ] Kindly use a better boilerplate for your HTML markup. I would suggest this option. [HTML boilerplate](https://www.freecodecamp.org/news/basic-html5-template-boilerplate-code-example/).
- [ ] You may then apply the class to your element correctly and then apply styles to this class in the stylesheet.
- [ ] Kindly give a descriptive name to your HTML file such as the `index.html` file. Your file at the moment is just a `.html`. Kindly name your file accordingly.